composer.json

```diff
@@ -12,13 +12,17 @@
     "autoload": {
         "psr-4": {
             "EzSystems\\EzPlatformMatrixFieldtypeBundle\\": "src/bundle/",
-            "EzSystems\\EzPlatformMatrixFieldtype\\": "src/lib/"
+            "EzSystems\\EzPlatformMatrixFieldtype\\": "src/lib/",
+            "Ibexa\\FieldTypeMatrix\\": "src/lib/",
+            "Ibexa\\Bundle\\FieldTypeMatrix\\": "src/bundle/",
+            "Ibexa\\Contracts\\FieldTypeMatrix\\": "src/contracts/",
+            "Ibexa\\Tests\\FieldTypeMatrix\\": "tests/"
```

Comment:
This should be moved into `autoload-dev` and also splited between `/tests/lib` and `/tests/bundle`